content/blog/2025/oddcontrast-guide.md

```diff
@@ -0,0 +1,180 @@
+- Hex
+- HSL
+- Lab
+- LCH
+- Oklab
+- Oklch
+- P3
+- sRGB
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2025/oddcontrast-formats.jpg',
+    alt: 'p3 is selected in the color format select field'
+  }]
+) }}
+
+Regardless of chosen color display format,
+you can use any valid CSS notation to enter foreground and background colors.
+For instance, you can use decimals in `oklch(.44 0.1 295)`
+even though the default is set to display the lightness value as a `%`.
+You can even used named colors! Here’s `honeydew` as the foreground color.
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2025/oddcontrast-named-colors-cropped.jpg',
+    alt: 'honeydew named colors is used as the foreground color'
+  }]
+) }}
+
+While not valid CSS notation, OddContrast also accepts 6 and 8-digit Hex
+colors without the `#`.
+This flexibility is helpful when coming from design tools as they
+often will not include the full color notation.
+
+## Workflow enhancements
+
+To streamline your color-editing tasks, OddContrast includes several helpful
+features. For starters, you can edit a single component of a color by using the
+visual sliders. These change depending on the color format you've selected. For
+instance, if Oklab is selected, the first slider controls the lightness (L).
+The next two sliders show options from green to red (A) and blue to yellow (B).
+The last slider sets the Alpha value and is consistent across all color formats.
+Additionally, you can use your keyboard arrow keys to nudge the color values
+incrementally when focused.
+
+Need to swap the foreground and background colors? No problem, select the left
+right arrow icon button or you can drag and drop the color swatches themselves.
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2025/oddcontrast-swaps.png',
+    alt: 'use the arrow icon to swap foreground and background colors'
+  }],
+  class='extend-large'
+) }}
+
+Once you are happy with your colors, you can easily copy your preferred color
+notation to the clipboard with a single click.
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2025/oddcontrast-clipboard.png',
+    alt: 'the copy to clipboard flow'
+  }]
+) }}
+
+## Visualize the in-gamut range
+
+OddContrast allows you to choose whether or not to display a color gamut for
+visual reference, and if so, which specific gamut you'd like to see. Once a
+gamut is selected, the color sliders will display the "in-gamut" boundaries and
+will update with the color values. To preview this, select the sRGB option in
+the "Show Gamut" field, edit the color sliders and see how every shift in color
+will affect the other channels.
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2025/oddcontrast-srgb-gamut.jpg',
+    alt: 'use the arrow icon to swap foreground and background colors'
+  }]
+) }}
+
+<div class="contain">
+{{ embed.figure(
+  data=[{
+    img: 'blog/2025/oddcontrast-gamut-warning.jpg',
+    alt: 'use the arrow icon to swap foreground and background colors'
+  }],
+    class='align-left'
```

Comment:
```suggestion
  class='align-left'
```

Reply:
Fixed in 613995ff159685af68a52ea345394d3d1f006710